content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md

```diff
@@ -45,11 +46,11 @@ This what is known to work with the code as of Developer Preview 2.-->
 
 ## Required CSDE Ports
 
-The current version of the [Corda combined worker](../cordapp-standard-development-environment/csde.html#gradle-helpers-for-the-combined-worker) runs on ports 7000, 5432, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 7000 for the system Control Center. The following section describes how to make the port available.
+The current version of the [Corda combined worker](../cordapp-standard-development-environment/csde.html#gradle-helpers-for-the-combined-worker) runs on ports 5005, 5432, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 5005 for the system Control Center. The following section describes how to make the port available.
```

Comment:
Is it definitely true that the port number for the system Control Center has changed?

Reply:
Confirmed with @manishr3 that the system Control Center port number has changed to 5005.